Assets/Scripts/EnemyController.cs

```diff
@@ -7,41 +7,54 @@
 
 public class EnemyController : MonoBehaviourPunCallbacks, IPunObservable
 {
+    [System.NonSerialized]
     public GameObject[] players;
+    [System.NonSerialized]
     public Transform player;
     private float distanceToPlayer;
-    public int minDistForMeleeAttack = 2;
-    public int minDistForMovement = 110;
-    
     private bool isAttackReady = true;
     private float attackAnimationDelay = 1.5f;
 
-    //DDA friendly variables--------------
-    public float attackDamage = 5f;
-    public int speed = 2;
+    [Header("DDA friendly variables - they might be changed by the DDAA")]
+    //the default values here should be used if DDA is not applied
+    public float meleeDamage = 90f;
+    public float projectileDamage = 30f;
+    public float speed = 3f;
     public float maxHealth = 50f;
-    public float health = 50f;
     public float shootingDistance = 25f;
+    public float minDistForMeleeAttack = 2;
+    [Tooltip("Stopping distance should be lower than minimum distance for melee")]
+    public float stoppingDistance = 2.5f;
+    public int minDistForMovement = 110;
     //-------------------------------------
+    [System.NonSerialized]
+    public float health = 50f;      //current player health
```

Comment:
if it means "current health", why not call it `currentHealth` and no comments would be needed? :D